### DISCUSSION TITLE
Загрузка и выгрузка таблицы для использования git

### DISCUSSION THREAD

**Discussion by mgmman** [totumonline/totum-ru-issues-and-discussions#425] [Как это сделать?] [answered]
Здравствуйте. У меня есть вопрос - как загрузить таблицу totum с сервера (я имею в виду всю таблицу, а не импорт/экспорт CSV) и как ее загрузить обратно (возможно на другой сервер)

**Comment by erpic-ru** [comment, marked as answer]
https://ru.docs.totum.online/export#intro

**Reply by ttm-alexeypolunin** [reply]
Только для git это абсолютно человеконечитаемая фигня. Когда-то была идея сделать формат выгрузки подходящий для отслеживания изменений в git, но от этого отказались из-за стоимости.
